src/common/cockpitpipe.c

```diff
@@ -1116,11 +1116,18 @@ cockpit_pipe_connect (const gchar *name,
   else
     {
       if (!g_unix_set_fd_nonblocking (sock, TRUE, NULL))
-        g_return_val_if_reached (NULL);
+        {
+          close (sock);
+          g_return_val_if_reached (NULL);
```

Comment:
We should fix the decorators or markup on ```g_return_val_xxx``` so that coverity thinks of them as ```no-return``` or ```abort()```.